### PR TITLE
update doc for memory management of prepared plan cache

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

As title. Some contents are updated in https://github.com/pingcap/docs/pull/8689. 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/9553
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: grafana-tidb-dashboard.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ To understand the key metrics displayed on the TiDB dashboard, check the followi
     - Execution Duration: the statistics of the execution time for SQL statements
     - Expensive Executor OPS: the statistics of the operators that consume many system resources per second, including `Merge Join`, `Hash Join`, `Index Look Up Join`, `Hash Agg`, `Stream Agg`, `Sort`, `TopN`, and so on
     - Queries Using Plan Cache OPS: the statistics of queries using the Plan Cache per second
+    - Plan Cache Miss OPS: the statistics of the number of times that the Plan Cache is missed per second.
```
Comment (Oreoxmt) [inline grafana-tidb-dashboard.md:86]:
```suggestion
    - Plan Cache Miss OPS: the statistics of the number of times that the Plan Cache is missed per second
```

File: sql-prepared-plan-cache.md [hunk 2]
```diff
@@ -123,6 +123,23 @@ MySQL [test]> select @@last_plan_from_cache;
...
+## Memory management of Prepared Plan Cache
+
+Using Prepared Plan Cache has some memory overhead. In internal tests, each cached plan consumes an average of 100 KiB of memory. Because Plan Cache is currently at the `SESSION` level, the total memory consumption is approximately `the number of sessions * the average number of cached plans in a session * 100 KiB`.
+
+For example, the current TiDB instance has 50 sessions in concurrency and each session has approximately 100 cached plans. The total memory consumption is approximately `50 * 100 * 100 KiB` = `512 MB`.
+
+You can control the maximum number of plans that can be cached in each session by configuring the system variable `tidb_prepared_plan_cache_size`. For different environments, the recommended value is as follows:
+
+- When the memory threshold of the TiDB server instance is <= 64 GiB, set `tidb_prepared_plan_cache_size` to `50`.
+- When the memory threshold of the TiDB server instance is > 60 GiB, set `tidb_prepared_plan_cache_size` to `100`.
```
Comment (Oreoxmt) [inline sql-prepared-plan-cache.md:135]:
```suggestion
- When the memory threshold of the TiDB server instance is > 64 GiB, set `tidb_prepared_plan_cache_size` to `100`.
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]

**Review by qw4990 — APPROVED** [review]
LGTM

**Pull request merged**